themes/components-training-theme/src/css/variables/_buttons.scss

```diff
@@ -0,0 +1,65 @@
+$btn-primary: () !default;
+$btn-primary: map-deep-merge(
+	(
+		background-color: $primary,
+		border-color: $primary,
+		color: $white,
+        border-radius: 0,
+		hover: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		focus: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		active: (
+			background-color: $primary-active,
+			color: $white,
+		),
+		disabled: (
+			background-color: $primary-disabled,
+		),
+	),
+	$btn-primary
+);
+
+$btn-outline-primary: () !default;
+$btn-outline-primary: map-deep-merge(
+    (
+        border-radius: 0,
```

Comment:
Same situation here, I would have saved a few lines if you went for reassigning the variable, rather than calling the entire mapping function.